package/SUMMARY.txt

```diff
@@ -29,6 +29,12 @@ or installed/upgraded via PyPI (using either `easy_install`_ or `pip`_)::
   easy_install -U MDAnalysis
```

Comment:
We can probably delete all references to `easy_install` when we go over this the next time round. I started cleaning it from wiki and other installation instructions.